### ISSUE TITLE
1.20.2 Autocrafting table crashes the game when in use

### ISSUE THREAD

**Issue by Yadokingu** [gnembon/carpet-autoCraftingTable#73]
> The game crashed whilst ticking block entity
> Error: java.lang.ClassCastException: class net.minecraft.class_8786 cannot be cast to class net.minecraft.class_3955 (net.minecraft.class_8786 and net.minecraft.class_3955 are in unnamed module of loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @52aa2946)

**Comment by Aculoss** [comment]
Can confirm.

**Comment by Astronomy20** [comment]
I can confirm too

**Comment by johnnygoboy** [comment]
Can this be updated? or is there a way to do it ourselves?

**Comment by Yadokingu** [comment]
> Can this be updated? or is there a way to do it ourselves?

Anybody can fork this. But you'd have to know Java to do the actual fixing.
I hope Gnembon will give us a final update for 1.20.2 and 1.20.3. I assume this addon will stop being developed once 1.21, with the official crafter, is out. Which is amazing and much better than the auto crafting table, btw.

**Comment by Aculoss** [comment]
I disagree, as the SS is not the same with the new auto crafter. Plus the locked slots still transmit a SS.

**Comment by Yadokingu** [comment]
> I disagree, as the SS is not the same with the new auto crafter. Plus the locked slots still transmit a SS.

You might not want to use cryptic short hand. Took me way too long to figure that one out. What does it matter that the two output different signal strength? Also I know that the new crafter is not the same. I already said it was way better. That implies it's different.
I assume we can thank Gnembon for it too, since it's suspiciously reminiscent to his prefered dropper crafting method.

**Comment by SirRemus** [comment]
I may have found a workaround until official release. [](https://github.com/SirRemus/carpet-autoCraftingTable/releases/tag/untagged-8245580005d261e59990)

**Comment by johnnygoboy** [comment]
What do i need to do to get the mod to work on the new version? I guess I can open the java files in notepad?

**Comment by SirRemus** [comment]
got a working version till the official release https://github.com/SirRemus/carpet-autoCraftingTable/releases/tag/1.20.2.

**Comment by johnnygoboy** [comment]
> got a working version till the official release https://github.com/SirRemus/carpet-autoCraftingTable/releases/tag/1.20.2.

awesome. thanks. I wish I knew how to fork it.

**Comment by BegleitPanzer** [comment]
> got a working version till the official release https://github.com/SirRemus/carpet-autoCraftingTable/releases/tag/1.20.2.

what was the problem with the current one

**Comment by johnnygoboy** [comment]
Can someone tell me how to update it to 1.20.4? Happy to do it myself if I know how.

**Comment by johnnygoboy** [comment]
Can it be updated to 1.20.4 - or can someone tell me how to do it?
